x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomLink/CustomLinkFlyout/helper.ts

```diff
@@ -73,7 +87,7 @@ export const DEFAULT_OPTION: FilterSelectOption = {
 
 export const filterSelectOptions: FilterSelectOption[] = [
```

Comment:
To stay consistent with the other compile time constants:

```suggestion
export const FILTER_SELECT_OPTIONS: FilterSelectOption[] = [
```